pkg/install/deployment.go

```diff
@@ -153,9 +153,9 @@ func WithDefaultSnapshotMoveData() podTemplateOption {
 	}
 }
 
-func WithDisableInformerCache() podTemplateOption {
+func WithEnableInformerCache() podTemplateOption {
 	return func(c *podTemplateConfig) {
-		c.disableInformerCache = true
+		c.enableInformerCache = true
```

Comment:
Shouldn't this be false ?

Reply:
@shubham-pampattiwar the param switches from "disable" to "enable" (so false is disabled), this func is called to *enable*, so it should be true.

Reply:
gotcha, thanks !